lib/reform/form/validation/unique_validator.rb

```diff
@@ -33,7 +38,12 @@ def validate_each(form, attribute, value)
     model = form.model_for_property(attribute)
 
     # search for models with attribute equals to form field value
-    query = model.class.where(attribute => value)
+    if options[:case_sensitive] == false
+      downcased_value = value.nil? ? value : value.downcase
+      query = model.class.where("lower(#{attribute}) = ?", downcased_value)
```

Comment:
Hah, I'd mistakenly assumed it could be parameterised the same way. That looks like a suitable solution; go for it =)

Interestingly, Rails 5 does that differently, using Arel. I don't know if/how that changes things though. https://github.com/rails/rails/blob/5-0-stable/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb#L446